omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,106 @@
+   can always find the original data by using the 'info' tab, as shown in the
+   :help:`OMERO.figure Help guide <figure.html#info>` (OMERO.figure supports a
+   complete figure creation workflow, including exporting figures into image
+   processing applications for final adjustments - see the
+   :help:`OMERO.figure Help guide <figure.html>` for full details).
+#. Having all the data belong to one user simplifies the UI experience for
+   public users. If necessary, ownership of data can be transferred using the
+   'Chown' privilege (see :doc:`restricted-admins` and
+   :doc:`/users/cli/chown`).
+
+Configuring OMERO.web
+---------------------
+
+#. Configure the :ref:`public_user`.
+#. Configure the OMERO.web :ref:`filter on the public user <public.url_filter>` by setting
+   :property:`omero.web.public.url_filter` to allow 'webclient' so that the
+   full webclient is visible for the public user, and thus the Data tree with
+   Projects and Datasets is also browsable, as well as the Tags tab and the
+   full image viewer.
+#. In order to have an automatically logged-in public user while still giving
+   your existing OMERO users an unchanged user experience (i.e. not
+   automatically logging them in as the public user), a dedicated,
+   :doc:`separate web server <unix/install-web/web-deployment>` for servicing
+   the public workflows can be added and configured to point at your existing
+   OMERO.server (in this case, the public user and the public URL filter need
+   to be set on this separate OMERO.web installation).
+
+Data configuration
+------------------
+
+#. Once the data is in the dedicated read-only group, it can be reorganized
+   and renamed to reflect the publication e.g. Projects can be renamed
+   according to the corresponding figure panels in the manuscript while the
+   names of the Datasets could be retained corresponding to different
+   treatment conditions represented in each figure panel. For example, Project
+   `Schleicher_etal_figure7_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27920>`_
+   contains images underlying the
+   `publication Figure panel 7c <http://rsob.royalsocietypublishing.org/content/royopenbio/7/11/170099/F7.large.jpg>`_.
+   Some Projects underlie two publication figure panels, such as Project
+   `Schleicher_etal_figure2_a_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27917>`_
+   where representative images are shown in panel a and the
+   corresponding quantification is shown in panel c of `Figure 2 <http://rsob.royalsocietypublishing.org/content/royopenbio/7/11/170099/F2.large.jpg>`_. 
+   This makes clear which original images are underlying which figure panels
+   in the publication.
+#. Data can also be tagged with OMERO tags to enhance the browsing
+   possibilities through these data for any user with basic knowledge of
+   OMERO. For example, see `Tag:Schleicher_etal_figure1_a <https://omero.lifesci.dundee.ac.uk/webclient/?show=tag-364188>`_). The
```

Comment:
The bracket-close at the end of the tag link seems out of place, there is no bracket open in the previous text.
![screen shot 2017-12-13 at 15 52 42](https://user-images.githubusercontent.com/2478303/33947982-b1cba870-e01d-11e7-835c-d319adf5f838.png)